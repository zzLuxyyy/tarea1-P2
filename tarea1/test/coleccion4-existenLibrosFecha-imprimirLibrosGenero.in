crearColeccion

crearFecha 01/01/1869
crearLibro 101112131 "Guerra y paz" "León" "Tolstói" "Épica historia sobre la nobleza rusa durante las guerras napoleónicas" 3
agregarEnColeccion

crearFecha 01/01/2006
crearLibro 141516171 "Cometas en el cielo" "Khaled" "Hosseini" "Una amistad marcada por la traición en el Afganistán contemporáneo" 1
agregarEnColeccion

crearFecha 19/10/1953
crearLibro 151617181 "Fahrenheit 451" "Ray" "Bradbury" "Un bombero quema libros en una sociedad donde leer es delito" 2
agregarEnColeccion

crearFecha 01/08/1965
crearLibro 161718192 "Dune" "Frank" "Herbert" "Una lucha política y espiritual en el desierto del planeta Arrakis" 5
agregarEnColeccion

crearFecha 17/09/1951
crearLibro 194512348 "Bestiario" "Julio" "Cortázar" "Siete cuentos extraños y perturbadores que exploran lo fantástico en lo cotidiano" 1
agregarEnColeccion

# No hay libros para la fecha
existenLibrosFechaColeccion 01/01/2025

# Hay libro para la fecha
existenLibrosFechaColeccion 19/10/1953

# No hay libros para el género
imprimirLibrosGeneroColeccion 13

# Hay 2 libros del género 1
imprimirLibrosGeneroColeccion 1

liberarColeccion

Fin