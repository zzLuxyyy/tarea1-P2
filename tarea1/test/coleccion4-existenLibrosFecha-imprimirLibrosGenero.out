1>2>3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>#  No hay libros para la fecha.
18>NO existen libros en la colección con fecha de edición 1/1/2025
19>#  Hay libro para la fecha.
20>Existen libros en la colección con fecha de edición 19/10/1953
21>#  No hay libros para el género.
22>23>#  Hay 2 libros del género 1.
24>Libro 141516171
Título: Cometas en el cielo
Autor: Khaled Hosseini
Descripción: Una amistad marcada por la traición en el Afganistán contemporáneo
Género: 1
Fecha de edición: 1/1/2006
Libro 194512348
Título: Bestiario
Autor: Julio Cortázar
Descripción: Siete cuentos extraños y perturbadores que exploran lo fantástico en lo cotidiano
Género: 1
Fecha de edición: 17/9/1951
25>26>Fin.
